### PR TITLE
Game logic

### PR DESCRIPTION
**Pull request by sgoddijn**

### CODE REVIEW

File: play.py [hunk 1]
```diff
@@ -63,25 +81,32 @@ def main(self):
...
+        for i in range(3, 0, -1):
+            self.initial_settlement_placements(i)
         return True
 
+    # This will place the initial settlement given the current player
+    def initial_settlement_placements(self, playerIndex):
+        player = self.players[playerIndex]
+        playerName = player.getName()
+        print("It is " + playerName + "\'s turn:")
+        possible_placement = self.game.getSettlementLocations(player)
```
Comment (noahjax) [inline play.py:98]: might have issue using get settlement locationsbecause that will eventually check if a road ends there. i can add a pregame bool to that function to handle it though

Comment (sgoddijn) [reply play.py:98]: Fair enough, I'll pass a boolean (firstTurn = true) and remove the road location

**Comment by noahjax** [comment]
how does the workflow go ẃhen i want to sign off but not merge yet?

-- Yeah, all I could see was the option to merge

File: play.py (hunk 1 above)
Comment (willlauer) [reply play.py:98]: Looks good to me.

**Pull request merged**